File: iniciante/javascript/1066.js
var input = require('fs').readFileSync('stdin', 'utf8')
var lines = input.split('\n')

let countEntradas = 0
let countPares = 0
let countImpares = 0
let countNegativos = 0
let countPositivos = 0

while(countEntradas++ < 5){
    let entrada = parseInt(lines.shift())
    if(entrada % 2 === 0){
        countPares++
    }
    if(entrada % 2 !== 0){
        countImpares++
    }
    if(entrada > 0){
        countPositivos++
    }
    if(entrada < 0){
        countNegativos++
    }

}

console.log(`${countPares} valor(es) par(es)`)
console.log(`${countImpares} valor(es) impar(es)`)
console.log(`${countPositivos} valor(es) positivo(s)`)
console.log(`${countNegativos} valor(es) negativo(s)`)

File: iniciante/python/1066.py
# -*- coding: utf-8 -*-

count = 0
entrada = []
count_par = 0
count_impar = 0
count_positivo = 0
count_negativo = 0

while count < 5:
    entrada.append(int(input()))
    if entrada[count] > 0:
        count_positivo += 1
    else:
        if entrada[count] != 0:
            count_negativo += 1
    if entrada[count] % 2 == 0:
        count_par += 1
    else:
        count_impar += 1

    count += 1

print(f'{count_par} valor(es) par(es)')
print(f'{count_impar} valor(es) impar(es)')
print(f'{count_positivo} valor(es) positivo(s)')
print(f'{count_negativo} valor(es) negativo(s)')
